### PR TITLE
Adds more wall types for faction camps central building

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary

SUMMARY: Content "Add log and rock wall types for faction camp central building."

#### Purpose of change

Finish adding the wall options for the central core of the faction camp.

#### Describe the solution

Added log and rock wall recipe and map support.

#### Describe alternatives you've considered

I forgot these options when I added the others, thanks to Murder Unicorn for pointing out they were missing.

#### Testing

Built full central camp buildings in each type to test them.

#### Additional context

These require an obscene amount of planks for the roof and floor constructions.  We may want to audit those recipes.  

log wall core
rock wall core

### CODE REVIEW

File: data/json/recipes/basecamps/recipe_modular_field_log.json [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmh_room0_log_northeast",
+    "blueprint_name": "northeast shack",
+    "blueprint_requires": [ { "id": "fbmh_0" } ],
+    "blueprint_provides": [ { "id": "fbmh_northeast" } ],
+    "blueprint_excludes": [ { "id": "fbmh_northeast" } ],
+    "blueprint_autocalc": true
```
Comment (ZhilkinSerg) [inline data/json/recipes/basecamps/recipe_modular_field_log.json:15]: I think `blueprint_autocalc` is not required anymore.

Comment (curstwist) [reply data/json/recipes/basecamps/recipe_modular_field_log.json:15]: I leave it out or add in something else?

Comment (jbytheway) [reply data/json/recipes/basecamps/recipe_modular_field_log.json:15]: Just omit it.  Any blueprint that lacks `blueprint_needs` and doesn't have `check_blueprint_needs: false` will have its needs auto-calculated.

Comment (curstwist) [reply data/json/recipes/basecamps/recipe_modular_field_log.json:15]: Will do, thanks.  I didn't see that mentioned in your PR.

Comment (jbytheway) [reply data/json/recipes/basecamps/recipe_modular_field_log.json]: Yeah, sorry.  Didn't mention it in the PR.  Was going to put it in the docs but then I didn't write them.

**Pull request merged**